src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/MicrosoftNetCoreAnalyzersResources.resx

```diff
@@ -1510,4 +1510,25 @@
     <value>and all other platforms</value>
     <comment>This call site is reachable on: 'windows' 10.0.2000 and later, and all other platforms</comment>
   </data>
+  <data name="PreferDictionaryContainsKeyCodeFixTitle" xml:space="preserve">
+    <value>Use 'ContainsKey(TKey)'</value>
```

Comment:
Prefer to have a simpler user-facing string for the code fix:
```suggestion
    <value>Use 'ContainsKey'</value>
```

Reply:
Fixed